Src/Types.cs

```diff
@@ -112,7 +112,7 @@ public static class ChannelStringList
 
             {Channel_t.CHAN_BATTERY, "BATTERY"},
             {Channel_t.CHAN_RESERVED, "RESERVED"},
-            {Channel_t.CHAN_MARKER, "MARKER"},
+            {Channel_t.CHAN_MARKER, "MARKERS"},
```

Comment:
why do you change this to `MARKERS` (with S)?

Reply:
MARKERS with S is correct with the output from API.